src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -63,6 +63,8 @@ public LauncherInitTask(final Stage newOwner, HostServices hostServices) {
      */
     @Override
     protected LauncherConfiguration call() {
+        // TODO: Use idiomatic JavaFX error handling.
```

Comment:
what does this entail?

Reply:
That actually means to use the `Task` API's own error handling methods, instead of using try-catch inside the `call()` function. This can be refactored with the help of `Task::setOnFailed`.